### ISSUE TITLE
[Feature Request] Sliding Time Window view for Record History

### ISSUE THREAD

**Issue by alaestor** [Razeeman/Android-SimpleTimeTracker#136]
### TL;DR
Record History window should have an option to view records from the last N hours.

### Current situation
The pane that shows record history only does so as a daily truncated view, meaning *you're only be able to see records that were created on the selected day* between 0~24hrs. The day can be changed by using the arrows, or selected by using the bottom center button to open a calendar.

### The 'Problem'
It can make viewing history at a glance more difficult for people with atypical schedules. For example: if I wake up at 8pm/20:00 for a night shift, *when the day rolls over after midnight, I'd have to press the left arrow to view 'yesterday'* in order to see the records from when my 'day' started. If there's a record that spans between the days, I have to either press it to see the proper duration, or flip back and forth and add up the times.

### Request
In addition to a truncated day view, I'd really like to see a sliding window view that shows all records for the last N hours. At least an option to *view records from the last 24 hours*. I'd suggest tab bar to the selection calendar, with tabs labeled "Day" for the current calendar UI (which should probably remain the default) and "Time" which has buttons for selecting, or dials for configuring, the timeframe for the sliding window. Ideally this would be a configurable number of hours, or have multiple choices (like 24, 48, 72).

#### P.s.
Admittedly, this is a very minor UX inconvenience, so I'd understand if this request is denied if implementation is non-trivial due to how your app is structured. I thank you for your time and consideration of this request.
